README.md

```diff
@@ -77,36 +89,32 @@ pip install torchserve torch-model-archiver
     source activate torchserve
     ```
 
+2. Optional if using torchtext models
+    ```bash
+    pip install sentencepiece
+    ```
+
 Now you are ready to [package and serve models with TorchServe](#serve-a-model).
 
 ### Install TorchServe for development
 
 If you plan to develop with TorchServe and change some of the source code, you must install it from source code.
 
-1. Install dependencies
+Please deactivate any conda env that you might be within.
+Run the following script from the top of the source directory.
 
-    ```bash
-    pip install psutil future -y
-    ```
-
-1. Clone the repo
-
-    ```bash
-    git clone https://github.com/pytorch/serve
-    cd serve
-    ```
-
-1. Make your changes executable
+NOTE: This script uninstalls existing `torchserve` and `torch-model-archiver` installations
 
-    ```bash
-    pip install -e .
-    ```
+#### For Debian Based Systems
+Verified on EC2 instances running Ubuntu DL AMI 28.x
 
-* To develop with torch-model-archiver:
+```
+./scripts/install_from_src_linux
```

Comment:
The name of the script file is incorrect. Please update to correct name `./scripts/install_from_src_ubuntu`

Reply:
fixed in a subsequent commit